README.md

```diff
@@ -954,60 +967,61 @@ Liquidity events are the usual way shareholders in private companies sell their
     improving the value of the company as a whole.
-  - But former employees and other shareholders often have difficulty initiating secondary
-    transactions as well.
+  - Former employees and other shareholders often have difficulty initiating secondary
+    transactions with a company.
     Private buyers may ask for the company’s internal financials in order to estimate the
     current and future value of its stock;
     the company may not wish to share this confidential information.
-    Companies
+  - Companies
     [also must consider](https://www.square1bank.com/insights/new-normal-secondaries-409a-valuation-process/)
     whether sales could influence their [409A valuation](#409a-valuations).
-  - Secondary sales are also an administrative and legal burden that may not make it to the
-    top of the list of priorities for busy startup CEOs and CFOs.
-  - In late-stage funding rounds, companies frequently do secondary offerings where employees
-    may be eligible to sell a portion of their stock.
-- 🔹 **Secondary buyers**: Thinking on all this
+  - Secondary sales are an administrative and legal burden that may not make it to the top of
+    the list of priorities for busy startup CEOs and CFOs.
+- 🔹 However, participation in the secondary market
   [has evolved](http://www.industryventures.com/2014/12/02/employee-liquidity-good-for-private-companies/)
-  [in recent years](https://techcrunch.com/2015/10/14/selling-private-company-shares-2-0/).
-  Some companies do see value in offering (mostly limited) opportunities for sale.
-  [SharesPost](http://sharespost.com/), [Equidate](https://www.equidateinc.com/), and
-  [EquityZen](https://equityzen.com/) have sought to establish a market around secondary sales,
-  particularly for well-known pre-IPO companies.
-  A few other secondary firms have emerged that have interest in certain purchases,
-  especially for larger sales from founders, early employees, or executives (parties
-  generally in possession of larger shares).
-  A company can work with the firm to facilitate multiple transactions.
-  These firms include [137 Ventures](http://137ventures.com/),
-  [ESO Fund](https://employeestockoptions.com/),
-  [Akkadian Ventures](https://www.akkadianventures.com/),
-  [Industry Ventures](http://www.industryventures.com/), [Atlas Peak](http://www.atlaspeakcap.com/),
-  and [Founders Circle](http://www.founderscircle.com/). The goal of these firms is to make
-  money of these purchases, so it will work if they believe the stock has chance of
-  significant future value.
-  In other words, if your company isn’t doing well, don’t be surprised if you can’t sell at
-  all on the secondary market.
+  [in recent years](https://techcrunch.com/2015/10/14/selling-private-company-shares-2-0/),
+  and more companies are seeing value in offering opportunities for sale or private equity
+  to [**secondary buyers**](https://en.wikipedia.org/wiki/Private_equity_secondary_market).
```

Comment:
FYI this wiki link is actually referring to private equity investments, a very different thing. Will revert.